src/components/MoneyRequestConfirmationList.js

```diff
@@ -173,6 +192,11 @@ function MoneyRequestConfirmationList(props) {
     const shouldCalculateDistanceAmount = props.isDistanceRequest && props.iouAmount === 0;
     const shouldCategoryEditable = !_.isEmpty(props.policyCategories) && !props.isDistanceRequest;
 
+    // Fetches the first tag of the policy
+    const tagKey = _.first(_.keys(props.policyTags));
+    const tag = lodashGet(props.policyTags, tagKey, {});
```

Comment:
Naming things is hard so I'm not 100% convinced this is best, but I think let's:
* rename `tagKey` to `tagListKey`
* rename `tag` to `tagList`